lib/index.js

```diff
@@ -272,10 +272,23 @@ internals.getParamsData = function (param, name, typeName) {
         };
     }
 
-    var type;
+    var originalType = param.type,
+        type;
     if (param.valids && param.valids.some(Joi.isRef)) {
         type = 'reference';
     }
+    else if (param.valids && param.valids.length === 1) {
+        // Simplify the single valid case into a constant display, i.e. {param: "value"}
+        type = internals.getExistsValues(type, param.valids);
+    }
+    else if (!name && param.valids) {
```

Comment:
The rendering of the template and the data used to make it still seems weird because of the following lines, can't we keep it in the templates elegantly ?